kubernetes.yaml

```diff
@@ -59,6 +61,9 @@ spec:
         - configMap:
             name: scrapyd-k8s-config
           name: scrapyd-k8s-config
+        #- name: joblogs
+        #  persistentVolumeClaim:
+        #    claimName: efs-claim
```

Comment:
Can we only include those things necessary running on minikube? I think EFS is AWS-specific, right?